scripts/apps/monitoring/tests/monitoring.spec.ts

```diff
@@ -218,6 +218,16 @@ describe('monitoring', () => {
             });
         }));
 
+        it('can get criteria for content type filter', inject((cards, session) => {
```

Comment:
These tests are only testing what query gets generated. The dropdown could be removed and the tests would still pass. Could you instead replace these with e2e tests where you would test if all content profiles appear in the dropdown and if correct items get filtered?

Reply:
Can you remove these since you added e2e tests?